### PR TITLE
Add my profile

### PR DESCRIPTION
**Pull request by ningke**
Add my mentor profile.

### CODE REVIEW

File: mentors/cpp/n.json [hunk 1]
```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username" : "ningke",
+    "name" : "Ning Ke",
+    "link_text" : "github profile",
```
Comment (Smarticles101) [inline mentors/cpp/n.json]: You might consider capitalizing github like GitHub

Comment (rpottsoh) [reply mentors/cpp/n.json]: Nice catch.  I agree.

Comment (rpottsoh) [reply mentors/cpp/n.json]: `Rakefile` isn't sensitive to `github`, just `Github`....  easily remedied....

Comment (rpottsoh) [reply mentors/cpp/n.json]: #791 created to guard against any form of miss-capitalized, but properly spelled, form of GitHub.

Comment (Smarticles101) [reply mentors/cpp/n.json]: Thanks @rpottsoh!

**Comment by rpottsoh** [comment]
@ningke when you have a moment, please make the changes to the way GitHub is capitalized for "link_text" as @Smarticles101 suggested and we'll get your bio merged.

**Comment by rpottsoh** [comment]
@ningke just a friendly reminder that "link_text" needs to be updated so that GitHub is spelled with proper capitalization.  Everything else looks fine.

**Comment by Smarticles101** [comment]
@rpottsoh seeing as @ningke hasn't responded in almost two weeks, I updated the capitalization so we can go ahead and get this merged.

**Pull request merged**